aspnetcore/mvc/razor-pages/razor-pages-authorization-conventions.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Razor Pages authorization conventions in ASP.NET Core
+author: guardrex
+description: Learn how to control access to pages with conventions at startup that authorize users and allow anonymous users to access individual pages or folders of pages.
+keywords: ASP.NET Core,Razor Pages,authorization,AllowAnonymousToPage,AllowAnonymousToFolder,AuthorizePage,AuthorizeFolder
+ms.author: riande
+manager: wpickett
+ms.date: 10/24/2017
+ms.topic: article
+ms.assetid: f65ad22d-9472-478a-856c-c59c8681fa71
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: mvc/razor-pages/razor-pages-authorization-conventions
+---
+# Razor Pages authorization conventions in ASP.NET Core
+
+By [Luke Latham](https://github.com/guardrex)
+
+One way to control access in your Razor Pages app is to use authorization conventions at startup. These conventions allow you to authorize users and allow anonymous users to access individual pages or folders of pages. The conventions described in this topic automatically apply [authorization filters](xref:mvc/controllers/filters#authorization-filters) to control access.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/mvc/razor-pages/razor-pages-authorization-conventions/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+## Require authorization to access a page
+
+Use the [AuthorizePage](/dotnet/api/microsoft.extensions.dependencyinjection.pageconventioncollectionextensions.authorizepage) convention via [AddRazorPagesOptions](/dotnet/api/microsoft.extensions.dependencyinjection.mvcrazorpagesmvcbuilderextensions.addrazorpagesoptions) to add an [AuthorizeFilter](/dotnet/api/microsoft.aspnetcore.mvc.authorization.authorizefilter) to the page at the specified path:
```

Comment:
Maybe a quick note somewhere on what this path means:

The path referenced here is the View Engine path. It's the Razor Pages root relative path without extension containing only forward slashes.

Reply:
I added a line to this effect to all four. The page ones say ...

> The specified path is the View Engine path, which is the Razor Pages root relative path without an extension and containing only forward slashes.

The folder ones say ...

> The specified path is the View Engine path, which is the Razor Pages root relative path.